frontend/app/controllers/Offer.scala

```diff
@@ -0,0 +1,11 @@
+package controllers
+
+import play.api.mvc.Controller
+
+trait Offer extends Controller {
+  def subscriber = NoCacheAction { implicit request =>
```

Comment:
I think it should be a `CachedAction`, there isn't anything on this page that requires user input.